proposals/NNNN-Dynamic-App-List-Sorting.md

```diff
@@ -74,12 +74,12 @@ Currently, there is no sequence provided that demonstrates how the icon is displ
 
 ## Potential downsides
 
-There is no potential downside since it is a modification to the current process of the SDL app icon display order.
+Since it is necessary to implement the large amount of modification due to implement multiple sorting methods, the developer must do the implementation carefully and also, it will take a long time.
 
 ## Impact on existing code
 
-The existing sorting process is left as it is. Furthermore, as processes/lists related to sorting are added, SDL Core is affected. Also, since the priority information of `apphmitype_sorting_list` is stored, the policy server is affected, too. However, I believe their changes are minor version changes because there are no changes in RPC and parameters.
+The existing sorting process is left as it is. Furthermore, as processes/lists related to sorting are added, SDL Core is affected. Also, since the priority information of `appHMIType_sorting_list` is stored, the policy server is affected, too. However, I believe their changes are minor version changes because there are no changes in RPC and parameters.
 
 ## Alternatives considered
 
-None.
+The alternative sorting method is user defined sort order. This sorting method would be a feature that some OEMs would want. However, since this specific sorting method would only be part of the HMI and have no other SDL component integration, we think it is difficult for SDL component to realize this function. Therefore, we think it is better this function to include into the HMI Integration Guidelines and SDL Core Guides documentation.
```

Comment:
>we think it is difficult for SDL component to realize this function.

Can you please clarify what you mean by "realize this function"?

Reply:
@theresalech -san,
Thank you for your review.
What I mean by "realize this function" is to add the user defined sort order function to the SDL Core.
Thank you.